datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/assertionUtils.tsx

```diff
@@ -18,7 +18,7 @@ export const sortAssertions = (a, b) => {
     if (!b.runEvents?.runEvents?.length) {
         return -1;
     }
-    return b.runEvents.runEvents[0].timestampMillis - a.runEvents.runEvents[0].timestampMillis;
+    return b.runEvents?.runEvents[0]?.timestampMillis - a.runEvents?.runEvents[0]?.timestampMillis;
```

Comment:
nice!

Reply:
I wonder if this should be copied to SaaS as well

Reply:
Done